docs/topics/proc_adding-source-platform-application.adoc

```diff
@@ -0,0 +1,46 @@
+:_newdoc-version: 2.15.0
+:_template-generated: 2024-2-21
+:_mod-docs-content-type: PROCEDURE
+
+[id="adding-source-platform-application_{context}"]
+= Adding a source platform in an application
+
```

Comment:
The 'Creating a source platform' section should come before all the other sections. The order of the other sections is fine.